pennylane/fermi/fermionic.py

```diff
@@ -324,6 +324,100 @@ def to_mat(self, n_orbitals=None, format="dense", buffer_size=None):
             wire_order=list(range(largest_order)), format=format, buffer_size=buffer_size
         )
 
+    def shift_operator(self, initial_position, final_position):
+        r"""Shifts an operator in the FermiWord from ``initial_position`` to ``final_position`` by applying the fermionic anti-commutation relations.
```

Comment:
The doc needs some improvement to render correctly:
1. A blank line between first and second sentences.
2. Using proper math block, see [here](https://docs.pennylane.ai/en/stable/_modules/pennylane/templates/subroutines/phase_adder.html#PhaseAdder) for example.
3. Proper citation with a link, see above example.